kitsune/messages/tasks.py

```diff
@@ -61,3 +64,55 @@ def _send_mail(locale):
         locale = settings.WIKI_DEFAULT_LANGUAGE
 
     _send_mail(locale)
+
+
+@shared_task
+def send_message(to, text=None, sender=None):
```

Comment:
For a celery task it would be more appropriate to pass the IDs instead of the objects and fetch a fresh copy from database when the task is running. If you pass an object, things might have changed by the time you run the task.